.github/workflows/pr.yml

```diff
@@ -15,6 +15,11 @@ jobs:
     if: ${{ github.repository == 'ni/measurementlink-labview' || github.event.pull_request.head.repo.full_name == 'ni/measurementlink-labview' }}
     uses: ./.github/workflows/run_g_tests.yml
 
+  run_vi_analyzer:
```

Comment:
@aepete, we're eventually going to want to run these tests (as well as the differ) on measlinkbot. Can you let @MounikaBattu17 what extra software needs to be installed to run VI Analyzer and the differ?

Reply:
Diff check requires no additional tooling -- just LabVIEW.

VI Analyzer check requires **VI Analyzer Toolkit 2020 64-bit**